File: sdkwork-ai-sdk-csharp/Models/DocumentResponse.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class DocumentResponse
    {
        public string? Title { get; set; }
        public string? Type { get; set; }
        public string? Status { get; set; }
        public Dictionary<string, object>? Metadata { get; set; }
        public string? DocumentId { get; set; }
        public string? KnowledgeBaseId { get; set; }
        public int? ChunkCount { get; set; }
        public int? CreatedAt { get; set; }
        public int? UpdatedAt { get; set; }
    }
}

File: sdkwork-ai-sdk-go/types/document_response.go
package types

// 文档响应
type DocumentResponse struct {
	Title string `json:"title"`
	Type string `json:"type"`
	Status string `json:"status"`
	Metadata map[string]interface{} `json:"metadata"`
	DocumentId string `json:"document_id"`
	KnowledgeBaseId string `json:"knowledge_base_id"`
	ChunkCount int `json:"chunk_count"`
	CreatedAt int `json:"created_at"`
	UpdatedAt int `json:"updated_at"`
}
